<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Evc05ChargerItem</class>
 <widget class="QWidget" name="Evc05ChargerItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="chargerItemWidget">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>0</y>
     <width>440</width>
     <height>230</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>23</y>
      <width>371</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="connectorInfoWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>107</width>
         <height>16777215</height>
        </size>
       </property>
       <widget class="QLabel" name="connectorId">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>23</y>
          <width>105</width>
          <height>71</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QLabel" name="connectorPower">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>130</y>
          <width>105</width>
          <height>51</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>220 V</string>
        </property>
       </widget>
       <widget class="QLabel" name="connectorType">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>95</y>
          <width>105</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Schuko</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="powerEnergyProgressBarWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <widget class="QWidget" name="powerAndEnergyWidget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>181</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#lineWidget {
    background-image: url(:/assets/images/rectangle.png);
    background-repeat : no-repeat;
    background-position: center;
}</string>
        </property>
        <widget class="QLabel" name="energyLabel">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>10</y>
           <width>81</width>
           <height>23</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">Energy</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="powerLabel">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>81</width>
           <height>23</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">Power</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QWidget" name="lineWidget" native="true">
         <property name="geometry">
          <rect>
           <x>73</x>
           <y>10</y>
           <width>21</width>
           <height>46</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>46</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="energyValue">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>33</y>
           <width>81</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="powerValue">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>33</y>
           <width>81</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="progressLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>%5</string>
        </property>
       </widget>
       <widget class="ProgressBarWidget" name="progressBar" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>65</y>
          <width>160</width>
          <height>16</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="socketIcon" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="reservationOrFaultedWidget" native="true">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>32</y>
     <width>391</width>
     <height>167</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>185</height>
    </size>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>391</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="iconLabelEmptyWidgetLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QWidget" name="emptyWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>125</width>
         <height>168</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="chargerItemNotificationIconAndLabelWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>168</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>35</y>
          <width>221</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="chargerItemNotificationIconAndLabelLayout">
         <item>
          <widget class="QLabel" name="reservationOrFaultedLabel">
           <property name="minimumSize">
            <size>
             <width>133</width>
             <height>0</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="chargerItemNotificationIconWidget" native="true">
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProgressBarWidget</class>
   <extends>QWidget</extends>
   <header>progressbarwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
